packages/govuk-frontend/src/govuk/errors/index.mjs

```diff
@@ -47,3 +47,25 @@ export class SupportError extends GOVUKFrontendError {
 export class ConfigError extends GOVUKFrontendError {
   name = 'ConfigError'
 }
+
+/**
+ * Indicates an issue with an element (possibly `null` or `undefined`)
+ */
+export class ElementError extends GOVUKFrontendError {
+  name = 'ElementError'
+
+  /**
+   * @param {Element} element - The element in error
+   * @param {object} options - options
+   * @param {string} options.componentName - The name of the component throwing the error
+   * @param {string} options.identifier - An identifier that'll let the user understand which element has an error (variable name, CSS selector)
+   * @param {typeof HTMLElement} [options.expectedType] - The type that was expected for the element
+   */
+  constructor(element, { componentName, identifier, expectedType }) {
```

Comment:
Thanks for the brill work in this PR @romaricpascal

One super tiny thing and we're good to go! If you turn on [`"strict": true`](https://www.typescriptlang.org/tsconfig#strict) it'll spot that we've got `@param` `[options.expectedType]` marked as optional but line 67 can crash when accessing `.name`

Optional is good though

If we default `expectedType = HTMLElement` we can remove all these lines:

```patch
--- A/packages/govuk-frontend/src/govuk/components/radios/radios.mjs
+++ B/packages/govuk-frontend/src/govuk/components/radios/radios.mjs
@@ -33,8 +33,7 @@
     if (!($module instanceof HTMLElement)) {
       throw new ElementError($module, {
         componentName: 'Radios',
+        identifier: '$module'
-        identifier: '$module',
-        expectedType: HTMLElement
       })
     }
```

Makes sense that `HTMLElement` is always the default expected type:

```suggestion
  constructor(element, { componentName, identifier, expectedType = HTMLElement }) {
```

Reply:
Updated it in the last push 🥳

Reply:
Cheers, approved it but remembered earlier feedback on destructuring in https://github.com/alphagov/govuk-frontend/pull/4104#discussion_r1327291883